index.html

```diff
@@ -78,15 +78,15 @@ <h1 class="solution-heading">Our Solutions</h1>
 
         <section class="the-team">
             <h1 class="team-heading">Meet the team</h1>
-            <span><hr class="horizontal-line"></span>
+            <hr class="horizontal-line">
             <div id="the-team-container">
                 
             </div>
         </section>
 
         <section class="partners">
             <h1 class="partners-heading">Our Partners</h1>
-            <span><hr class="horizontal-line"></span>
+            <hr class="horizontal-line">
             <ul class="partners-list">
                 <li><a href=""><img class="partners-logo" src="images/logos/kadict-logo.png" alt=""></a></li>
                 <li><a href=""><img class="partners-logo"  src="images/logos/airbnb.png" alt=""></a></li>
```

Comment:
- [ ] [Optional] For this section on both the _**Home**_ and _**About**_ Page, kindly use the design template as a guideline to list partners for the events.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/181916849-f2279653-fa4e-4868-ba19-629d44672889.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/181916807-135a372f-ab94-4c47-970b-2c1236b9b43a.png)